templates/instance/admin.rs.html

```diff
@@ -33,7 +33,7 @@ <h1>@i18n!(ctx.1, "Administration")</h1>
       <label for="long_description">@i18n!(ctx.1, "Long description")<small>@i18n!(ctx.1, "Markdown syntax is supported")</small></label>
       <textarea id="long_description" name="long_description">@Html(form.long_description)</textarea>
 
-      @input!(ctx.1, default_license (text), "Default license", form, errors, "minlenght=\"1\"")
+      @input!(ctx.1, default_license (text), "Default content license", form, errors, "minlenght=\"1\"")
```

Comment:
The message in the .pot file is "Default article license", you should either add this one to the .pot or change "content" for "article". https://github.com/Plume-org/Plume/blob/master/po/plume.pot#L361